packages/client/components/ScopePhaseAreaGitLab.tsx

```diff
@@ -0,0 +1,62 @@
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {ScopePhaseAreaGitLab_meeting$key} from '../__generated__/ScopePhaseAreaGitLab_meeting.graphql'
+import ScopePhaseAreaAddGitLab from './ScopePhaseAreaAddGitLab'
+
+interface Props {
+  isActive: boolean
+  gotoParabol: () => void
+  meetingRef: ScopePhaseAreaGitLab_meeting$key
+}
+
+graphql`
+  fragment ScopePhaseAreaGitLab_teamMember on TeamMember {
+    integrations {
+      gitlab {
+        auth {
+          provider {
+            scope
```

Comment:
-1 scope isn't ever used in the component. I think we're requesting this in order to see if there is a provider. If that's the case, you can grab the `id` or `__typename` field to express that (since those fields are automatically requested & immutable)